platform/include/retry_utils.h

```diff
@@ -78,9 +94,9 @@ void RetryUtils_ParamsReset( RetryUtilsParams_t * pRetryParams );
  * must use this function between retry failures to add exponential delay.
  * This function will block the calling task for the current timeout value.
  *
- * @param[in, out] pRetryParams structure containing retry parameters.
+ * @param[in, out] pRetryParams Structure containing retry parameters.
  *
- * @return true after successful sleep, false when all attempts are exhausted.
+ * @return True after a successful sleep, false when all attempts are exhausted.
  */
 bool RetryUtils_BackoffAndSleep( RetryUtilsParams_t * pRetryParams );
```

Comment:
Per offline conversation, the return value of this function is not a predicate and we should use an enum instead for the status.  In a separate PR I will remove the use of bool in this file throughout the code.